setup.py

```diff
@@ -12,4 +12,5 @@
       url="https://github.com/tlsfuzzer/tlsfuzzer",
       description="TLS test suite and fuzzer.",
       license="GPLv2",
+      install_requires=["ecdsa >= 0.15", "tlslite-ng >= 0.8.1"],
```

Comment:
requirements.txt are 0.8.2, .1 is too old; and we don't need to explicitly depend on ecdsa; that's picked through tlslite-ng...

Reply:
> requirements.txt are 0.8.2, .1 is too old;

Oh, I copied 0.8.1 from README. I'll change it then.

> and we don't need to explicitly depend on ecdsa; that's picked through tlslite-ng...

We do, because it's also a direct dependency: https://github.com/tlsfuzzer/tlsfuzzer/blob/b5790435bd52caddc30ab6e465e33fcb23c49985/scripts/rsa-keys-generator.py#L13.

Reply:
I've updated the version of `tlslite-ng` here via a force-pushed commit. Kept `ecdsa` per above.